xml/glossary.xml

```diff
@@ -962,6 +962,13 @@
    </para>
   </glossdef>
  </glossentry>
+  <glossentry xml:id="gloss.hammer"><glossterm>SLEShammer</glossterm>
+  <glossdef>
+   <para>
+    When you first boot a node in Cloud via PXE, it is booted with the SLEShammer image. This performs the initial hardware discovery, and  registers the node with crowbar. Once you allocate the node, it is rebooted with a regular SLES installation image.
```

Comment:
Please use entities: When you first boot a node in &productname; via PXE, it is booted with the SLEShammer image. This performs the initial hardware discovery, and registers the node with &crow;.